linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -300,10 +308,13 @@ impl LineraNet for LocalKubernetesNet {
 }
 
 impl LocalKubernetesNet {
+    #[allow(clippy::too_many_arguments)]
     fn new(
         network: Network,
         testing_prng_seed: Option<u64>,
         binaries: BuildArg,
+        no_build: bool,
+        docker_image_name: String,
```

Comment:
If this is `&str` (I don't think you _need_ ownership) wouldn't it make life easier?

All the futures are joined at the end of the function so there are no borrows which will outlive the reference.